doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1,1736 +1,1323 @@
-Local Definition}.
-
-\begin{ErrMsgs}
-\item \errindex{{\ident} already exists}
-\end{ErrMsgs}
-
-\begin{Variants}
-\item {\tt Let {\ident} : {\term$_1$} := {\term$_2$}.}
-\item {\tt Let Fixpoint {\ident} \nelist{\fixpointbody}{with} {\tt .}.}
-\item {\tt Let CoFixpoint {\ident} \nelist{\cofixpointbody}{with} {\tt .}.}
-\end{Variants}
-
-\SeeAlso Sections \ref{Section} (section mechanism), \ref{Opaque},
-\ref{Transparent} (opaque/transparent constants), \ref{unfold} (tactic
-    {\tt unfold}).
-
-%%
-%% Inductive Types
-%%
-\subsection{Inductive definitions
-\index{Inductive definitions}
-\label{gal-Inductive-Definitions}
-\comindex{Inductive}
-\label{Inductive}
-\comindex{Variant}
-\label{Variant}}
-
-We gradually explain simple inductive types, simple
-annotated inductive types, simple parametric inductive types,
-mutually inductive types. We explain also co-inductive types.
-
-\subsubsection{Simple inductive types}
+Section :ref:`Typed-terms`.
+
+.. cmd:: Definition @ident := @term.
+
+   This command binds :token:`term` to the name :token:`ident` in the environment,
+   provided that :token:`term` is well-typed.
+
+.. exn:: @ident already exists
+
+.. cmdv:: Definition @ident : @term := @term.
+
+   It checks that the type of :token:`term`:math:`_2` is definitionally equal to
+   :token:`term`:math:`_1`, and registers :token:`ident` as being of type
+   :token:`term`:math:`_1`, and bound to value :token:`term`:math:`_2`.
+
+
+.. cmdv:: Definition @ident {* @binder } : @term := @term.
+
+   This is equivalent to ``Definition`` :token:`ident` : :g:`forall`
+   :token:`binder`:math:`_1` … :token:`binder`:math:`_n`, :token:`term`:math:`_1` := 
+   fun :token:`binder`:math:`_1` …
+   :token:`binder`:math:`_n` => :token:`term`:math:`_2`.
+
+.. cmdv:: Local Definition @ident := @term.
+
+   Such definitions are never made accessible through their
+   unqualified name by Import and its variants (see :ref:`Import`).
+   You have to explicitly give their fully qualified name to refer to them.
+
+.. cmdv:: Example @ident := @term.
+
+.. cmdv:: Example @ident : @term := @term.
+
+.. cmdv:: Example @ident {* @binder } : @term := @term.
+
+These are synonyms of the Definition forms.
+
+.. exn:: The term @term has type @type while it is expected to have type @type
+
+See also Sections :ref:`Opaque,Transparent,unfold`.
+
+.. cmd:: Let @ident := @term.
+
+This command binds the value :token:`term` to the name :token:`ident` in the
+environment of the current section. The name :token:`ident` disappears when the
+current section is eventually closed, and, all persistent objects (such
+as theorems) defined within the section and depending on :token:`ident` are
+prefixed by the let-in definition ``let`` :token:`ident` ``:=`` :token:`term`
+``in``. Using the ``Let`` command out of any section is equivalent to using
+``Local Definition``.
+
+.. exn:: @ident already exists
+
+.. cmdv:: Let @ident : @term := @term.
+
+.. cm
```

Comment:
It seems to me that this should not start with a capital letter, or be reformulated to be moved above.